### ISSUE TITLE
Accessibility Audit for Facility Locator

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#12524]
## Description
This epic will track any accessibility issues, failures, or improvements that should be considered for the new redesign of Facility Locator.

**Comment by kmariepat-cityfriends** [comment]
Next steps: talk to @laflannery about which cards are aligned to the scope of Jordan's design work. Will do at end of March after designs are complete and we move into design accessibility review

**Comment by Agile6MSkinner** [comment]
Moved issues to other epics and closed for consolidation and tidiness
